samples/logging/prometheus-grafana-opentelemetry/Core_8/Endpoint/SomeCommandHandler.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+using System.Threading.Tasks;
+using NServiceBus;
+using NServiceBus.Logging;
+
+class SomeCommandHandler :
+    IHandleMessages<SomeCommand>
+{
+    static ILog log = LogManager.GetLogger<SomeCommandHandler>();
+    static Random random = new Random();
+
+    public async Task Handle(SomeCommand message, IMessageHandlerContext context)
+    {
+        await Task.Delay(random.Next(50, 250), context.CancellationToken)
+            .ConfigureAwait(false);
```

Comment:
I thought we no longer use ConfigureAwait in sample code?

Reply:
We still do in all samples, as we have not removed them yet. That's a good point though, so I'll remove it from here.